public/app/themes/clarity/src/components/c-article-item/view-blog-feed.php

```diff
@@ -49,15 +53,13 @@
     <div class="content">
         
         <h1>
-            <a href="<?php echo esc_url(get_permalink($id)); ?>"><?php echo $post['title']['rendered']; ?></a>
+            <a href="<?php echo esc_url($link); ?>"><?php echo $post->post_title; ?></a>
```

Comment:
Maybe we could use: `get_the_title($id)`

This function will apply sanitation including language support, if needed.